src/events/event_handler.ts

```diff
@@ -136,26 +136,26 @@ export class eventHandler {
 				* Get the Relative Path of each file using repoPath
 				* Get the fileID using relative_path found in above step
 		*/
+
 		// For the current open repoPath, get the repo_id and from config File
 		const configJSON = readYML(this.settings.CONFIG_PATH);
```

Comment:
Can you check ConfigUtils class, that reads this YML file for us and we can access the data through object attribute

Reply:
Having a look